celery_slack/attachments.py

```diff
@@ -60,6 +61,10 @@ def get_task_prerun_attachment(task_id, task, args, kwargs, **cbkwargs):
     if cbkwargs["flower_base_url"]:
         attachment["attachments"][0]["title_link"] = cbkwargs["flower_base_url"] + "/task/{tid}".format(tid=task_id)
 
+    if cbkwargs["show_custom_username"]:
+        username = cbkwargs.get("username", "{}@{}".format(platform.node(), task.name))
+        attachment["username"] = username
```

Comment:
Is this the username of the slack user? Would it be more appropriate to do something like `Celery@<node>` since the task name is already shown in every task-related message? That way a custom username would show up as `MyUsername@<node>`.

Reply:
I did it that way because when you have a lot of messages from different workers and servers it is faster to read one line with all the info than reading the slack username + message info.

And it is only the default behavior. If you pass `username="some_user"` the slack username will be `some_user` not `some_user@task_name`. If you want to change the default username I have no problem.